### PR TITLE
[4.0] Correct use of strong and em

### PR DESCRIPTION
**Pull request by brianteeman**
Replace the outdated b and i tags with strong and em. As of html5 b and i  have a different meaning now

### CODE REVIEW

File: administrator/components/com_contenthistory/tmpl/preview/preview.php [hunk 1]
```diff
@@ -37,7 +37,7 @@
 			<?php foreach ($value->value as $subName => $subValue) : ?>
 				<?php if ($subValue) : ?>
 					<tr>
-						<td><i>&nbsp;&nbsp;<?php echo $subValue->label; ?></i></td>
+						<td><em>&nbsp;&nbsp;<?php echo $subValue->label; ?></em>/td>
```
Comment (Quy) [inline administrator/components/com_contenthistory/tmpl/preview/preview.php]:
```suggestion
						<td><em>&nbsp;&nbsp;<?php echo $subValue->label; ?></em></td>
```

**Comment by brianteeman** [comment]
thanks for the review @quy I was writing this on a train watching my battery run out

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 49754cf3a83b22558b1f26b1fe65fb632fa43d9a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24903">issues.joomla.org/tracker/joomla-cms/24903</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**